index.html

```diff
@@ -4,484 +4,528 @@
+
+          <!-- Flying icons -->
+          <div class="graphics__flying-icon google-icon">
+            <img
+              class="google-img"
+              src="./src/icons/flat-color-icons_google.svg"
+              alt="google icon"
+            />
+          </div>
+
+          <div class="graphics__flying-icon paypal-icon">
+            <img
+              class="paypal-img"
+              src="./src/icons/simple-icons_paypal.svg"
+              alt="paypal icon"
+            />
+          </div>
+
+          <div class="graphics__flying-icon indeed-icon">
+            <img
+              class="indeed-img"
+              src="./src/icons/simple-icons_indeed.svg"
+              alt="indeed icon"
+            />
+          </div>
+
+          <div class="graphics__flying-icon slack-icon">
+            <img
+              class="slack-img"
+              src="./src/icons/grommet-icons_slack.svg"
+              alt="slack icon"
+            />
+          </div>
+
+          <!-- First flying card -->
+          <div class="graphics__flying-card first-flying-card">
+            <div class="flying-card__img-block first-img">
+              <div class="flying-card__circle">
+                <img src="./src/icons/star.svg" alt="star" />
+                <p class="flying-card__rating">4.8</p>
               </div>
+            </div>
+            <p class="flying-card__title">Maliha</p>
+            <p class="flying-card__subtitle">UX Designer</p>
+          </div>
 
-              <!-- Second flying card -->
-              <div class="right-wrapper__flying-card second-flying-card">
-                <div class="flying-card__img-block second-img">
-                  <div class="flying-card__circle">
-                    <img src="./src/icons/star.svg" alt="star" />
-                    <p class="flying-card__rating">4.5</p>
-                  </div>
-                </div>
-                <h2 class="flying-card__title">Adam</h2>
-                <p class="flying-card__subtitle">Project Manager</p>
+          <!-- Second flying card -->
+          <div class="graphics__flying-card second-flying-card">
+            <div class="flying-card__img-block second-img">
+              <div class="flying-card__circle">
+                <img src="./src/icons/star.svg" alt="star" />
+                <p class="flying-card__rating">4.5</p>
               </div>
             </div>
+            <p class="flying-card__title">Adam</p>
+            <p class="flying-card__subtitle">Project Manager</p>
           </div>
         </div>
       </section>
-    </header>
 
-    <main>
-      <!-- First section -->
-      <section class="first-section">
-        <div class="container first-section__container">
-          <h2 class="first-section__title">Popular Job Posts</h2>
-          <!-- Cards -->
-          <div class="first-section__cards-wrapper">
+      <!-- Job posts -->
+      <section class="job-posts">
+        <div class="container job-posts__container">
```

Comment:
```suggestion
        <div class="container">
```

Move styles from `.job-posts__container` to `.job-posts` and remove this class as it is unused.